collectors/pod.go

```diff
@@ -324,13 +338,22 @@ func (pc *podCollector) collectPod(ch chan<- prometheus.Metric, p v1.Pod) {
 				c.Name, nodeName)
 		}
 
+		if gpu, ok := req[v1.ResourceNvidiaGPU]; ok {
```

Comment:
According to https://github.com/kubernetes/kubernetes/blob/master/pkg/api/validation/validation_test.go#L3295-L3329, it seems that apiserver allows both request for gpu not setted and setted.